src/cljs_time/internal/unparse.cljs

```diff
@@ -50,8 +50,18 @@
    (fn [s d]
      (unparse-period s d (.getDate d) min max))))
 
+(defn unparse-day-of-week
+  ".getDay returns 0-6, shifts to 1-7"
+  ([min] (unparse-day-of-week min min))
+  ([min max]
+   (fn [s d]
+     (let [raw-day-of-week (.getDay d)
+           day-of-week (if (= raw-day-of-week 0)
+                         7 raw-day-of-week)]
+       (unparse-period s d day-of-week min max)))))
+
 (defn unparse-day-of-year
-  ([min] (unparse-day min min))
+  ([min] (unparse-day min min)) ;; <- this right?
```

Comment:
It is right, but I can see how it looks a bit weird 😉 

Maybe the 1-ary versions of these functions is a bit superfluous?

Reply:
What caught my eye was that `unparse-day-of-year` calls `unparse-day`. Turns out `unparse-day-of-year` is not even used.

The bigger issue is that parsing and unparsing of `day-of-year` is actually wrong. (i.e. the "D" fields). This is currently matched to day-of-month ("d"). 

Happy to fix that in another PR though.

Reply:
Hah, interesting. Seems like an oversight when I rewrote the parser.

Reply:
You mind just getting rid of this comment and I'll get this merged?

Reply:
Done